easybuild/easyconfigs/a/ABINIT/ABINIT-7.10.4-intel-2015a-incl-deps.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '7.10.4'
+versionsuffix = '-incl-deps'
+
+homepage = 'http://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy, charge density and
+ electronic structure of systems made of electrons and nuclei (molecules and periodic solids) within Density Functional
+ Theory (DFT), using pseudopotentials and a planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = ['http://ftp.abinit.org/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['bfc76a5f93c3f148c049f57541864e76']
+
+configopts = "--with-mpi-prefix=$EBROOTIMPI/intel64 --with-trio-flavor='etsf_io+netcdf' "
```

Comment:
we only have `$MPI_LIB_DIR` and `$MPI_INC_DIR`, so no, for now

what we would need here is `$MPI_BIN_DIR`, and then we can use `$MPI_BIN_DIR/../` reliably here